docs/motif_liquidator.md

```diff
@@ -0,0 +1,330 @@
+## Motif Liquidator: High Performance DNA Motif Analysis
```

Comment:
Please remove this file from the PR. This documentation is specific to an unfinished branch motif_liquidator_matrix and the functionality is not present in master (and likely never will be).

Reply:
All set.